src/Security/Voter/Committee/AdministrateCommitteeFeedItemVoter.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AppBundle\Security\Voter\Committee;
+
+use AppBundle\Committee\CommitteePermissions;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CommitteeFeedItem;
+use AppBundle\Security\Voter\AbstractAdherentVoter;
+
+class AdministrateCommitteeFeedItemVoter extends AbstractAdherentVoter
```

Comment:
Can you add a test suite for this Voter please ?

Reply:
Done